### PR TITLE
Gokhan Lesson 23 

### PR DESCRIPTION
**Pull request by G-Memis**

### CODE REVIEW

File: Lesson 23 Gokhan/styles.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
+* {
+    margin: 0;
```
Comment (annasvst) [inline Lesson 23 Gokhan/styles.css]: It's usually not good (and not efficient) to remove margin and padding from all elements at once. Better to remove margin and padding selectively from elements that really don't need it (e.g. body, ul, ol, etc.)

File: Lesson 23 Gokhan/styles.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    /* fallback for old browsers */
+    background: -webkit-linear-gradient(to right, #3b8d99, #6b6b83, #aa4b6b);
+    /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to right, #3b8d99, #6b6b83, #aa4b6b);
+    /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+
+}
+
+.accordion-container {
+    padding: 3rem;
```
Comment (annasvst) [inline Lesson 23 Gokhan/styles.css]: If you want to use rem, make sure that you set font-size explicitly on the body / html/ root element.

**Review by annasvst — CHANGES_REQUESTED** [review]
Hi Gökhan, all good code wise (I left minor comments for the future). However, I'd like you to correct the folder name. Folder should be named exactly 'Lesson23' and then there should be folder 'gokhan' inside of it. There you can place html, css and js files. :) 

After you move the files, you should be able to commit and push again. If you have problems, let me know.

File: Lesson 23 Gokhan/styles.css (hunk 2 above)
Comment (G-Memis) [reply Lesson 23 Gokhan/styles.css]: i changed it with 20px :)

**Review by annasvst — APPROVED** [review]

**Pull request merged**